lighthouse-core/test/results/sample_v2.json

```diff
@@ -670,6 +670,18 @@
         ]
       }
     },
+    "jankless-font": {
+      "id": "jankless-font",
+      "title": "Fonts with `font-display: optional` are preloaded",
+      "description": "Preloading optional fonts can reduce layout shifts and improve CLS. [Learn More](https://web.dev/optimize-cls/#web-fonts-causing-foutfoit)",
+      "score": null,
+      "scoreDisplayMode": "notApplicable",
+      "details": {
+        "type": "table",
+        "headings": [],
+        "items": []
```

Comment:
wanna tweak the test/results/artifacts/artifacts.json so that this audit has results here?

Reply:
Yes, I'll make sure to to do that!